test/rpc/test_deps.py

```diff
@@ -98,3 +98,20 @@ def test_rpc_deps_git(project_root, profiles_root, dbt_profile, unique_schema):
     }]
     deps_with_packages(packages, bad_packages, project_root, profiles_root, unique_schema)
 
+
+@pytest.mark.supported('postgres')
+def test_rpc_deps_git_subdir(project_root, profiles_root, dbt_profile, unique_schema):
+    # TODO: Change me to something that fishtown-analytics manages!
+    # here I just moved the dbt_utils code into a subdirectory
+    packages = [{
+        'git': 'https://github.com/dmateusp/dbt-utils.git',
+        'revision': 'dmateusp/0.5.0/move_dbt_utils_to_subdir',
+        'subdirectory': 'dbt_projects/dbt_utils'
+    }]
```

Comment:
Nice! Ok, here are some options:
1. Use a long-lived subdirectory of `dbt-utils`:
```python
    packages = [{
        'git': 'https://github.com/dmateusp/dbt-utils.git',
        'revision': '0.5.0',
        'subdirectory': 'integration_tests'
    }]
```
But because `integration_tests` has a local dependency on `../`, we'd need to add a dummy `dbt_project.yml` file to the directory right above this one. (Those relative paths become relative to the root project location.)

2. Use a less-well established "package" that has a _lot_ of project subdirectories (and which would really benefit from the addition of this feature):
```python
packages = [{
        'git': 'https://github.com/fishtown-analytics/dbt-labs-experimental-features',
        'subdirectory': 'materialized-views'
    }]
```

3. We could add a branch to `dbt-utils`, similar to `move_dbt_utils_to_subdir`... and hope it's never accidentally deleted.

Reply:
Sorry I didn't get if these options were for me or @kwigley ^^, I'd use `dbt-labs-experimental-features` on a specific commit sha, then we wouldn't worry about anything breaking the tests :)